docs/mods/suite/Quests/Developer/Quests/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating Quests
+sidebar_position: 1
+---
+
+:::info
+To create a quest you will first need to create a chapter, a quest is always created inside a chapter and a chapter can
+be put inside a chapter group.
+
+For details on creating a chapter, see the [Chapters](../Chapters/index.md) page.
+:::
+
+To create a quest, right anywhere in the blank space of the quest book and select the type of quest you want to create.
```

Comment:
Strictly, the type of _task_ you want to create... it creates a default quest with one task added to it.